source/Calamari/Kubernetes/Integration/AzureCli.cs

```diff
@@ -92,5 +93,10 @@ public void ConfigureAksKubeCtlAuthentication(Kubectl kubectlCli, string cluster
 
             kubectlCli.ExecuteCommandAndAssertSuccess("config", "set-context", clusterName, $"--namespace={@clusterNamespace}");
         }
+
+        public static void SetConfigDirectoryEnvironmentVariable(IDictionary<string, string> environmentVars, string workingDirectory)
+        {
+            environmentVars.Add("AZURE_CONFIG_DIR", Path.Combine(workingDirectory, "azure-cli"));
+        }
```

Comment:
@johnsimons Made this a static functionality so it's at least the directory name won't get out of sync